subprojects/files/src/test/groovy/org/gradle/api/internal/file/collections/DirectoryWalkerTest.groovy

```diff
@@ -29,7 +29,10 @@ import java.util.concurrent.atomic.AtomicInteger
 class DirectoryWalkerTest extends AbstractDirectoryWalkerTest<DirectoryWalker> {
     @Override
     protected List<DirectoryWalker> getWalkers() {
-        return [new DefaultDirectoryWalker(), new ReproducibleDirectoryWalker()]
+        return [
+            new DefaultDirectoryWalker(TestFiles.fileSystem()),
```

Comment:
was calling a non-existent default constructor, without initializing the file system parameter